frontend/gateway/gateway.go

```diff
@@ -361,7 +361,7 @@ func (b *bindMount) Mount() ([]mount.Mount, func() error, error) {
 	return []mount.Mount{{
 		Type:    "bind",
 		Source:  b.dir,
-		Options: []string{"bind", "ro"},
+		Options: []string{"bind", "ro", "nosuid", "nodev", "noexec"},
```

Comment:
maybe `relatime` as well to reduce random writes due to the access time updates and therefore better I/O latencies?

Reply:
This is mounted readonly, so I don't think it would do anything.

Reply:
Ah right